test-harness/modules/mocker/common.py

```diff
@@ -53,6 +53,13 @@ def _create_common_endpoints():
         res.append(ReflectingTcpIpEndpoint(ip='127.0.0.1', port=8181))
         # mesos
         res.append(MesosEndpoint(ip='127.0.0.2', port=5050))
+        # additional mesos instances for leader DNS changes tests
+        res.append(MesosEndpoint(ip='127.0.0.3', port=5050))
+        # mesos non-leading master that runs Mesos and AR
+        res.append(ReflectingTcpIpEndpoint(ip='127.0.0.10', port=5050))
+        res.append(ReflectingTcpIpEndpoint(ip='127.0.0.10', port=80))
```

Comment:
Then let's remove the other two - they are used for marathon leader testing if I remember correctly.

Adjusting the tests should not be hard.